repository-data/application/src/main/resources/hcm-config/namespaces/hee/report.yaml

```diff
@@ -32,22 +32,14 @@ definitions:
             hipposysedit:path: hee:summary
             hipposysedit:primary: false
             hipposysedit:type: Text
-            hipposysedit:validators: [optional]
+            hipposysedit:validators: [required]
           /contentBlocks:
             jcr:primaryType: hipposysedit:field
             hipposysedit:multiple: true
             hipposysedit:ordered: true
             hipposysedit:path: hee:contentBlocks
             hipposysedit:type: hippo:compound
-            hipposysedit:validators: [contentblocks-validator]
-          /relatedContent:
```

Comment:
I understand that `relatedContent` field isn't mentioned in the spec, but did we agree with Jon on this just to be sure ?